### PR TITLE
implementing grammar suggestions from issue #715

### PR DESCRIPTION
**Pull request by em-bellis**

### CODE REVIEW

**Review by Teebusch — APPROVED** [review]

**Comment by Teebusch** [comment]
This is excellent! Thank you so much, @em-bellis!

The changes look good to me, but perhaps a native English speaking maintainer should review & merge.

**Comment by mondorescue** [comment]
The text reads much better with the edits. I'll merge this PR.

**Pull request merged**
